mne_bids/read.py

```diff
@@ -9,6 +9,7 @@
 import os.path as op
 import glob
 import json
+import re
```

Comment:
as Alex shows in his comment: https://github.com/mne-tools/mne-bids/pull/607/files#r513752558

the "re" module is not necessary for what you want to achieve

```suggestion
```